### DISCUSSION TITLE
Copy paste not working correctly

### DISCUSSION THREAD

**Discussion by espi66** [musescore/MuseScore#16907] [General]
When I copy-paste a verse with several instruments, the pasted version loses many if not all the instrument voices.
All you can see is the pasted notes but with no sound when they are played. This is something close to a show stopper!

This bug wasn't in 4.0.1, but in 4.0.2 (even in the nightly build) it is there are reproducible very easily.

Workaround to this is copy (ctrl-x) the beats that don't make a sound and paste it back (ctrl-v) to where they already were. This helps but does not stop from the bug happening all over again.

You have done a great job with musescore! However this error made me want to downgrade back to 4.0.1, but that isn't easily available anymore. Please flag this one urgent.

**Comment by shoogle** [comment]
Thanks for the positive feedback!

Please report the problem with copy & paste at https://github.com/musescore/MuseScore/issues
